takeoffs-and-landings/src/main/java/io/spine/example/airport/tl/supplies/SuppliesEventConsumer.java

```diff
@@ -102,6 +103,7 @@ public void onNext(SuppliesEvent event) {
             EventMessage eventMessage = (EventMessage) unpack(event.getPayload());
             context.emittedEvent(eventMessage, actorContext);
         }
+        // #enddocfragment "onNext"
```

Comment:
Same as above.

Reply:
Same here, because of several `@Override` annotations.